ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestConfig.java

```diff
@@ -602,6 +610,12 @@
             DefaultErrorMessages.class,
             Importance.LOW,
             KSQL_SERVER_ERRORS_DOC
+        ).define(
+            KSQL_CONNECT_SERVER_ERROR_HANDLER,
```

Comment:
Let's make add this new config into KsqlConfig instead of KsqlRestConfig. KsqlRestConfig is meant for configs relevant to ksql's REST server. The new config being added more configures the behavior of ksql's engine, rather than its REST server.

Reply:
Done in 4d4ea68.